Verification/VerificationClass.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+class VerificationClass {
+
+	private $integer;
+	private $string;
+	private $float;
+	private $array;
+	
+	/**
+	 * 変数の初期化だけ行う
+	 */
+	function __construct() {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Visibility must be declared on method "__construct"
* Opening brace should be on a new line